### PR TITLE
feature: use new achievement text mappings

### PR DESCRIPTION
**Pull request by dhenkel92**
# Based on PR #496

### CODE REVIEW

File: src/components/achievements/modals/AchievementModal.tsx [hunk 1]
```diff
@@ -228,7 +226,7 @@ const AchievementModal: React.FC<AchievementModalProps> = ({
                                                         <IndicatorBar
                                                             maxSteps={maxSteps}
                                                             currentStep={currentStep}
-                                                            progressDescription={progressDescription}
+                                                            progressDescription={footer}
```
Comment (Jonasdoubleyou) [inline src/components/achievements/modals/AchievementModal.tsx:229]: rename also in this component?

File: src/config.ts [hunk 2]
```diff
@@ -21,4 +21,4 @@ export const DD_ENV = window.liveConfig?.RUNTIME_DD_ENV ?? process.env.REACT_APP
 
 export const TALKJS_APP_ID = window.liveConfig?.RUNTIME_TALKJS_APP_ID ?? process.env.REACT_APP_TALKJS_APP_ID;
 
-export const GAMIFICATION_ACTIVE = (window.liveConfig?.RUNTIME_GAMIFICATION_ACTIVE ?? 'false') === 'true';
+export const GAMIFICATION_ACTIVE = (window.liveConfig?.RUNTIME_GAMIFICATION_ACTIVE ?? 'true') === 'true';
```
Comment (Jonasdoubleyou) [inline src/config.ts:24]: Is "true" a good default? Maybe we should just activate in dev deployments / locally using the respective ENVs?

**Review by Jonasdoubleyou — APPROVED** [review]

File: src/config.ts (hunk 2 above)
Comment (dhenkel92) [reply src/config.ts:24]: 🙄 No it's not. Very good catch!

This was never supposed to be comitted.

File: src/components/achievements/modals/AchievementModal.tsx (hunk 1 above)
Comment (dhenkel92) [reply src/components/achievements/modals/AchievementModal.tsx:229]: I would say that it's okay in this case. For an Indicator bar it's actually a progress description and we are just using the footer to save the text.

**Pull request merged**